assembly/frame/executive.ts

```diff
@@ -138,10 +138,9 @@ export namespace Executive{
      * @param source source of the transaction (external, inblock, etc.)
      * @param utx transaction
      */
-    export function validateTransaction(utx: ISignedTransaction): u8[] {
+    export function validateTransaction(utx: ISignedTransactionType): u8[] {
```

Comment:
Shouldn't we use `ISignedTransaction`?

Reply:
We have to provide types to ISignedTransaction? f.e ISignedTransaction<T, S, A, N>